css-four-column-layout/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Four Column Layout</title>
+  <link href="styles.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+<div class="container">
+  <div class="row header-row">
+    <header class="header">How It Works</header>
+  </div>
+  <div class="row">
+    <div class="four-phone-pictures-div">
+      <div class="column">
+        <img src="images/screen-1.png" alt="screen-1" class="phone-pic">
+        <img src="images/switch-icon.png" alt="switch-icon" class="icon">
+      </div>
+      <div class="column">
+        <img src="images/screen-2.png" alt="screen-2" class="phone-pic">
+        <img src="images/filter-icon.png" alt="filter-icon" class="icon">
+      </div>
+      <div class="column">
+        <img src="images/screen-3.png" alt="screen-3" class="phone-pic">
+        <img src="images/slider-icon.png" alt="slider-icon" class="icon">
+      </div>
+      <div class="column">
+        <img src="images/screen-4.png" alt="screen-4" class="phone-pic">
+        <img src="images/bars-icon.png" alt="bars-icon" class="icon">
+      </div>
+    </div>
+  </div>
+  <div class="row footer-row">
+    <div class= "footer-column one">
+      <h3>App Switching</h3>
+      <p>Toggles all your apps on and off.</p>
+    </div>
+    <div class="footer-column two">
+      <h3>Trip Filtering</h3>
+      <p>Evaluates your trip requests.</p>
+    </div>
+    <div class="footer-column three">
+      <h3>Service Filters</h3>
+      <p>Accepts only your ideal trips.</p>
+    </div>
+    <div class="footer-column four">
+      <h3>Stats + Mileage</h3>
+      <p>Tracks your trips across all apps.</p>
+    </div>
+  </div>
```

Comment:
I can see why you broke these pieces of the layout into rows based on how they look in terms of "layout", but this makes the content of each column seem separate from one another. You should have one `row` with four `columns`, each of which holds the image, icon, and text for that column

Reply:
revised!